concreate/test/collector.py

```diff
@@ -0,0 +1,65 @@
+import os
+import logging
+import shutil
+import subprocess
+
+
+logger = logging.getLogger('concreate')
+
+
+class TestCollector(object):
+    def __init__(self, descriptor_dir, target_dir):
+        self.descriptor_dir = os.path.abspath(descriptor_dir)
+        self.target_dir = os.path.abspath(target_dir)
+        self.test_dir = os.path.join(self.target_dir, 'test')
+
+        # remove old test so we can get fresh ones collected
+        if os.path.exists(self.test_dir):
+            shutil.rmtree(self.test_dir)
+        os.makedirs(self.test_dir)
+
+    def fetch_steps(self, version):
+        """Methods fetches builtin steps """
+        logger.info("Fetching common steps")
```

Comment:
What does this message mean? Can you rewrite it?

Reply:
You cannot for now. I was thinking about making it overridable via .concreate. do you thin its needed?
I'll probably expand log message about steps git url.

Reply:
I was commenting just the message itself.

But you highlighted a good point. Please open a ticket on how to add a story for custom steps or how  to modify current ones.